web/app/webpack.config.js

```diff
@@ -24,7 +24,13 @@ module.exports = {
         exclude: /node_modules/,
         use: [
           'babel-loader',
-          { loader: 'eslint-loader', options: { fix: true } }
+          {
+            loader: 'eslint-loader',
```

Comment:
Could we configure this behavior by environment? I'd like to make sure CI breaks when eslint is failing.

Reply:
I took a stab at doing this; it required separating out our dev and prod webpack configs, which one we use now depends on `NODE_ENV`.

Reply:
While I support splitting them out, I'm not sure it is required *quite* yet. Why not just:

```
options: {
  fix: true,
  emitWarning: process.env.NODE_ENV !== 'production'
```